packages/faucet/package.json

```diff
@@ -18,16 +18,17 @@
     "build:rules": "firebase-bolt database-rules.bolt"
   },
   "dependencies": {
-    "@celo/contractkit": "1.0.0-beta3",
-    "@celo/utils": "1.0.0-beta3",
+    "@celo/connect": "1.0.2",
```

Comment:
We have contractkit v1.1.0-beta.1 out right now, should the versions refer to that instead of 1.0.2?

Reply:
I think that the beta3 was a clutch. I prefer to have all our tools that required a fix version with stable ones.
This particular type issue, was because it was missing the @celo/connect lib. I updated the others but it wasn't really required.